ttnn/cpp/ttnn/operations/sliding_window/sliding_window.cpp

```diff
@@ -4,8 +4,48 @@
 
 #include "sliding_window.hpp"
 
+namespace ttnn::operations::sliding_window{
+    std::size_t SlidingWindowConfig::get_hash() const {
+        return std::hash<std::string>{}(std::to_string(batch_size_)
```

Comment:
you can simply use:
```
return std::hash<std::string>()(to_string());
```
instead of duplicating it.
Also please move the `to_string()` impl to cpp from hpp.

Reply:
Will do.

Reply:
Fixed.